File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/Scimitar.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Sword;

/* JADX INFO: loaded from: classes3.dex */
public class Scimitar extends Sword {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.weapon_sword_scimitar_name;
        this.idDescription = R.string.weapon_sword_scimitar_description;
        this.idImage = R.drawable.scimitar;
        this.price = 41L;
        this.constitution = 6;
        this.dexterity = 2;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/Scimitar.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Sword

class Scimitar : Sword() {
    override fun configureProperties() {
        idName = R.string.weapon_sword_scimitar_name
        idDescription = R.string.weapon_sword_scimitar_description
        idImage = R.drawable.scimitar
        price = 41L
        constitution = 6
        dexterity = 2
    }
}
